src/event_loop.rs

```diff
@@ -175,16 +179,21 @@ impl Default for ControlFlow {
 }
 
 impl EventLoop<()> {
-    /// Alias for `EventLoopBuilder::new().build()`.
+    /// Alias for `EventLoopBuilder::new().build().unwrap()`.
     #[inline]
     pub fn new() -> EventLoop<()> {
+        EventLoopBuilder::new().build().unwrap()
+    }
+
+    /// Alias for `EventLoopBuilder::new().build()`.
+    pub fn new_result() -> Result<EventLoop<()>, CreationError> {
```

Comment:
This would be called `try_new` - though I think it's pretty redundant to have, people should just use `EventLoopBuilder::new().build()`

Reply:
Removed